config/config.go

```diff
@@ -173,8 +174,25 @@ func GetLaunchTimeout() time.Duration {
 	return duration
 }
 
+// GetStopTimeout returns the grace period time for a pod to die
+// Returns time in seconds as an integer
 func GetStopTimeout() int {
-	return GetConfig().GetInt(COMPOSE_STOP_TIMEOUT)
+	// the timeout value is expressed as duration in string format
+	valStr := GetConfig().GetString(COMPOSE_STOP_TIMEOUT)
```

Comment:
we can use use directly getDuration right since viper supports it?

Reply:
GetDuration() internally casts various data types. For example, it accepts both 25 and "25", but outputs 25ns. 
Since we now supports only duration format for timeout, the error handling (getduration returns no error, it is ignored iternally) based on zero value will be tricky. Error handling is required to return default timeout. @kmala what do you think?

Reply:
The value will have to be validated somewhere. It's probably easier in this case to use `GetConfig().GetDuration(key)` because it let's you offload the parsing then you can validate the value later.

For example:

"0m" and "0s" would both evaluate to a "0s" duration, which you can validate afterwards by checking that the duration is 0.

Reply:
@ridv If the input is specified with unit of time ("25s"), then we're good and requires no more validation.
As part of this jira, we have changed the format of this timeout from integer to duration type (25 to 25s). We now disallow timeout to be specified as 25 or "25" (25 is just an example). GetDuration accepts both 25 and "25" and treats them as value in nanoseconds and conversion to seconds gives 0 as the value. If the value is coming as 0 seconds, we again need to return the default. Let me make this change.